tests/ops/op_math/test_controlled.py

```diff
@@ -1053,13 +1053,14 @@ def test_non_differentiable_one_qubit_special_unitary(self):
     def test_differentiable_one_qubit_special_unitary(self):
         """Assert that a differentiable qubit special unitary uses the zyz decomposition."""
 
-        op = qml.ctrl(qml.RZ(qml.numpy.array(1.2), 0), (1))
+        op = qml.ctrl(qml.RZ(qml.numpy.array(1.2), 0), (1, 2, 3, 4))
```

Comment:
Instead of modifying this test to be the "more than one control" case, can we add a new test and have on test for each case? One test for a single control wire, and one test for multiple control wires?